include/wlr/render/egl.h

```diff
@@ -46,37 +46,25 @@ void wlr_egl_finish(struct wlr_egl *egl);
  */
 bool wlr_egl_bind_display(struct wlr_egl *egl, struct wl_display *local_display);
 
-/**
- * Refer to the eglQueryWaylandBufferWL extension function.
- */
-bool wlr_egl_query_buffer(struct wlr_egl *egl, struct wl_resource *buf,
-	EGLint attrib, EGLint *value);
-
 /**
  * Returns a surface for the given native window
  * The window must match the remote display the wlr_egl was created with.
  */
 EGLSurface wlr_egl_create_surface(struct wlr_egl *egl, void *window);
 
-/**
- * Creates an egl image from the given client buffer and attributes.
- */
-EGLImageKHR wlr_egl_create_image(struct wlr_egl *egl,
-		EGLenum target, EGLClientBuffer buffer, const EGLint *attribs);
-
 /**
  * Creates an egl image from the given dmabuf attributes. Check usability
  * of the dmabuf with wlr_egl_check_import_dmabuf once first.
  */
 EGLImageKHR wlr_egl_create_image_from_dmabuf(struct wlr_egl *egl,
-		struct wlr_dmabuf_buffer_attribs *attributes);
+	struct wlr_dmabuf_buffer_attribs *attributes);
```

Comment:
I wonder if we should move this into `gles2/texture.c`, just like the wl-drm one.

Reply:
I don't feel strongly either way.

Reply:
I'm anticipating a bit wrt. Vulkan and WSI here - we can just let it as is for now and see what happens with `wlr_egl` in the future.

Your idea about merging `wlr_egl` and `wlr_renderer` could solve it too.

Reply:
I retract my idea of merging wlr_egl and wlr_renderer.